tools/lsp/completion.rs

```diff
@@ -301,14 +300,17 @@ pub(crate) fn completion_at(
     } else if node.kind() == SyntaxKind::Document {
         let r: Vec<_> = [
             // the $1 is first in the quote so the filename can be completed before the import names
```

Comment:
that comment is now outdated.  And I still think we should keep the old way so we first complete the filename and then the Component you want to import, otherwise the components won't get auto-completion.